cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -304,6 +305,17 @@ func (c *CVCController) getVolumePolicy(
 	return volumePolicy, nil
 }
 
+// isReplicaAffinityEnabled checks if replicaAffinity has been enabled using
+// cstor volume policy
+func (c *CVCController) isReplicaAffinityEnabled(cvc *apis.CStorVolumeClaim) bool {
+	policyName := cvc.Annotations[string(apis.VolumePolicyKey)]
+	volumePolicy, err := c.getVolumePolicy(policyName, cvc)
```

Comment:
why are doing getVolumePolicy multiple times?

Reply:
We can pass the policy as an arg to caller , will change that

Reply:
Some changes are not part of this PR, as they are coming with reconcile changes and default policy changes

Reply:
Done, removed the redundant call in this PR itself.